tests/browser/snapshots/storybook-docs/Docs-Components-IdleTimeout-Docs-prop-table-matches-snapshot-1.txt

```diff
@@ -17,12 +17,12 @@
   [
     "formatMessage",
     "A formatting function that returns the string to be used in the warning modal. The formatting function is provided the timeTilTimeout (in minutes).\n\n(timeTilTimeout: number) => ReactNode",
-    "(timeTilTimeout: number): React.ReactNode => { const unitOfTime = timeTilTimeout === 1 ? 'minute' : 'minutes'; return ( <p> You&apos;ve been inactive for a while. <br /> Your session will end in{' '} <strong> {timeTilTimeout} {unitOfTime} </strong> . <br /> <br /> Select &quot;Continue session&quot; below if you want more time. </p> ); }"
+    "(timeTilTimeout: number): React.ReactNode => { const unitOfTime = timeTilTimeout === 1 ? t('idleTimeoutDialog.min') : t('idleTimeoutDialog.mins'); return ( <p> {t('idleTimeoutDialog.messageLine1')} <br /> {t('idleTimeoutDialog.messageLine2')} <strong> {timeTilTimeout} {unitOfTime} </strong> . <br /> <br /> {t('idleTimeoutDialog.continueSessionMessage')} </p> ); }"
```

Comment:
I never noticed that the contents of this function were spat out in this table 😵

Reply:
Not something to address right now